parser/src/grammar.lalrpop

```diff
@@ -108,19 +109,19 @@ TypeBinding: TypeBinding<Id> = {
         TypeBinding {
             comment: None,
             name: id.clone(),
-            alias: Alias::new(id, params, Type::variant(row)),
+            typ: Type::forall(params, Type::variant(row)),
```

Comment:
I'd rather not lose the information of `id` here. During typechecking, when `id` is refereed to, the `typ/alias` field is loaded and for efficiency and the sake of user-friendliness returning a `Type::Alias` is going to give more information than `Type::Forall` (though they should typecheck the same),